doc/changes/changes_1.2.0.md

```diff
@@ -0,0 +1,17 @@
+# JUnit5 System Extensions 1.2.0, released 2019-04-22
+
+Code name: Mute output
```

Comment:
Choose a different code name, this one is already used.

Reply:
Fixed.